test/test-utils.ts

```diff
@@ -135,6 +139,7 @@ function pipInstall() {
     '--disable-pip-version-check',
   ]);
   if (proc.status !== 0) {
+    console.log('' + proc.stderr);
```

Comment:
Please let me know if I should remove the console line. Think it helps checking the error

Reply:
Tiny Q, are you concatenating to convert to a string? What does it look like before this?

Reply:
`proc.stderr` gives a buffer, sth like `EE AB 6A 5C ...` that sort of thing, `'' + ` converts to meaningful string and preserves all the line breaks. I actually wanted originally to do:

```
if (proc.status !== 0) {
  throw new Error(
      'Failed to install requirements with pip.' +
        ' venv = ' +
        ' details: ' +
        proc.stderr
  )
}
```

but it doesn't preserve the line break and the printout gives only confusion 😂